RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.cpp

```diff
@@ -120,38 +163,48 @@ StatusCode AugmentClustersFCCee::execute([[maybe_unused]] const EventContext &ev
     sumEnLayer.assign(numLayersTotal, 0.);
     double phiMin = 9999.;
     double phiMax = -9999.;
+    int module_id_Min = 1536;
```

Comment:
this should not be hardcoded. It's a property of the detector geometry, I think you should be able to retrieve it from the geometry service - or even the readout